easel/lib/screens/creator_hub/creator_hub_screen.dart

```diff
@@ -319,6 +387,31 @@ class _CreatorHubContentState extends State<CreatorHubContent> {
       ),
     );
   }
+
+  Widget buildSelectedBox({required String title, required CreatorHubViewModel viewModel, required Color color, required CollectionType collectionType}) {
```

Comment:
`required CreatorHubViewModel viewModel` No need passing view Model

Reply:
I needed this view model in the method, so rather than passing context and getting the view model from context, I did pass the view model directly. 



<img width="1309" alt="image" src="https://user-images.githubusercontent.com/115459890/204503972-8c96a617-7a3d-4710-b248-477c0a37cfb9.png">

Reply:
why not use context.read() here?